test/integration/targets/azure_rm_dnsrecordset/tasks/main.yml

```diff
@@ -158,6 +158,22 @@
     that: 
       - results.changed
 
+- name: create TXT records in a new record set
+  azure_rm_dnsrecordset:
+    resource_group: "{{ resource_group }}"
+    relative_name: "_txt.{{ domain_name }}.com"
+    zone_name: "{{ domain_name }}.com"
+    record_type: TXT
+    state: present
+    records:
+      - entry: "v=spf1 a -all"
```

Comment:
can you add more than one entry to test it?

Reply:
ack, will extend the test case.

Reply:
Added additional entries.